DJ ERROR 2024-01-21 12:50:32,916 log Internal Server Error: /
Traceback (most recent call last):
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/sync.py", line 534, in thread_handler
    raise exc_info[1]
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = await get_response(request)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/core/handlers/base.py", line 253, in _get_response_async
    response = await wrapped_callback(
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/sync.py", line 479, in __call__
    ret: _R = await loop.run_in_executor(
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/current_thread_executor.py", line 40, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/sync.py", line 538, in thread_handler
    return func(*args, **kwargs)
  File "/home/kriyanshi/mercury-2.3.7/mercury/server/urls.py", line 20, in index
    return render(request, "index.html")
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
DJ ERROR 2024-01-21 12:50:32,930 runserver [35;1mHTTP GET / 500 [0.25, 127.0.0.1:37750][0m
DJ ERROR 2024-01-21 13:16:15,935 log Internal Server Error: /
Traceback (most recent call last):
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/sync.py", line 534, in thread_handler
    raise exc_info[1]
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = await get_response(request)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/core/handlers/base.py", line 253, in _get_response_async
    response = await wrapped_callback(
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/sync.py", line 479, in __call__
    ret: _R = await loop.run_in_executor(
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/current_thread_executor.py", line 40, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/sync.py", line 538, in thread_handler
    return func(*args, **kwargs)
  File "/home/kriyanshi/mercury-2.3.7/mercury/server/urls.py", line 20, in index
    return render(request, "index.html")
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
DJ ERROR 2024-01-21 13:16:15,941 runserver [35;1mHTTP GET / 500 [0.05, 127.0.0.1:39516][0m
DJ ERROR 2024-01-21 13:16:16,998 log Internal Server Error: /
Traceback (most recent call last):
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/sync.py", line 534, in thread_handler
    raise exc_info[1]
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = await get_response(request)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/core/handlers/base.py", line 253, in _get_response_async
    response = await wrapped_callback(
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/sync.py", line 479, in __call__
    ret: _R = await loop.run_in_executor(
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/current_thread_executor.py", line 40, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/sync.py", line 538, in thread_handler
    return func(*args, **kwargs)
  File "/home/kriyanshi/mercury-2.3.7/mercury/server/urls.py", line 20, in index
    return render(request, "index.html")
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
DJ ERROR 2024-01-21 13:16:17,004 runserver [35;1mHTTP GET / 500 [0.05, 127.0.0.1:39516][0m
DJ ERROR 2024-01-21 13:16:17,204 log Internal Server Error: /
Traceback (most recent call last):
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/sync.py", line 534, in thread_handler
    raise exc_info[1]
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = await get_response(request)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/core/handlers/base.py", line 253, in _get_response_async
    response = await wrapped_callback(
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/sync.py", line 479, in __call__
    ret: _R = await loop.run_in_executor(
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/current_thread_executor.py", line 40, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/asgiref/sync.py", line 538, in thread_handler
    return func(*args, **kwargs)
  File "/home/kriyanshi/mercury-2.3.7/mercury/server/urls.py", line 20, in index
    return render(request, "index.html")
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/kriyanshi/mercury-2.3.7/menv/lib/python3.10/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
DJ ERROR 2024-01-21 13:16:17,210 runserver [35;1mHTTP GET / 500 [0.04, 127.0.0.1:39516][0m
